package.json

```diff
@@ -14,7 +14,6 @@
     "build": "fedx-scripts webpack",
     "prepare": "husky install",
     "i18n_extract": "BABEL_ENV=i18n fedx-scripts babel src --quiet > /dev/null",
-    "is-es5": "es-check es5 ./dist/*.js",
```

Comment:
Perhaps is still worthwhile to include here?

Reply:
Nah. Looks like [it was removed from the template ](https://github.com/openedx/frontend-template-application/commit/51a147fc872231f80b8f3762754aff91b2773eaf) at the same time the CI job was.